calico_upgrade/tests/st/calico_upgrade/test_crud.py

```diff
@@ -0,0 +1,1995 @@
+# Copyright (c) 2015-2017 Tigera, Inc. All rights reserved.
```

Comment:
quick glance through these, I don't see any convert-specific actions on top of the calicoctl base.  what's the state of these STs?